resources/migrations/001-create-bookmarks.edn

```diff
@@ -0,0 +1,2 @@
+{:up ["CREATE TABLE IF NOT EXISTS bookmarks (id int, title text, tags text, created_at timestamp);"]
```

Comment:
I'd argue using seperate `.sql` files for the following reason:
- Native syntax highlighting. This should usually catch small syntax errors that you might make.
- Easier to write longer queries, imo.

I'll leave it to other folks to decide on this.